### ISSUE TITLE
Build failed: com.android.ide.common.process.ProcessException: Failed to execute aapt

### ISSUE THREAD

**Issue by bencmack** [wmcmahan/react-native-calendar-events#179]
### Environment

 SDKs:
      iOS SDK:
        Platforms: iOS 11.4, macOS 10.13, tvOS 11.4, watchOS 4.3
      Android SDK:
        Build Tools: 23.0.1, 25.0.0, 25.0.1, 25.0.2, 26.0.1, 26.0.2, 26.0.3, 28.0.1
        API Levels: 23, 25, 26, 28
    IDEs:
      Android Studio: 3.1 AI-173.4819257
      Xcode: 9.4.1/9F2000 - /usr/bin/xcodebuild
    npmPackages:
      react: 16.4.1 => 16.4.1 
      react-native: 0.56.0 => 0.56.0 
    npmGlobalPackages:
      react-native-cli: 2.0.1
      react-native-git-upgrade: 0.2.7

### Steps to Reproduce

Install library -> follow ReadMe instructions -> rebuild project on Android (react-native run-android)

### Expected Behavior

Project should build successfully and display in emulator.

### Actual Behavior

Project build fails. Error message is as follows:

`:react-native-calendar-events:processReleaseResources FAILED

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':react-native-calendar-events:processReleaseResources'.
> com.android.ide.common.process.ProcessException: Failed to execute aapt`

Uninstalled the library. Build was successful. Reinstall the library, build fails again. Tried many solutions I found online all to no avail.

**Comment by jeveloper** [comment]
Any luck?

**Comment by bencmack** [comment]
Yes! I had to set the buildToolsVersion and such within android/build.gradle folder of the library in node_modules equal to those in my project's android/build.gradle.

If anyone else is experience this, what worked for me was first checking build.gradle in myApp/android/build.gradle. At the bottom of that file should be something like this:
ext {
    buildToolsVersion = "26.0.3"
    minSdkVersion = 16
    compileSdkVersion = 26
    targetSdkVersion = 26
    supportLibVersion = "26.1.0"
}

Then, in myApp's node_module folder, find the folder for "react-native-calendar-events" (you'll have to scroll a lot) and within that folder navigate to android/build.gradle where you'll see something like this:
android {
    compileSdkVersion 26
    buildToolsVersion "26.0.3"

    defaultConfig {
        minSdkVersion 16
        targetSdkVersion 26
        versionCode 1
        versionName "1.0"

    }
} 

When I set all of the versions in myApp/node_modules/react-native-calendar-events/android/build.gradle to be the same as those in myApp/android/build.gradle everything worked as expected!

This trick helped me with a few other libraries as well.
